R/build.r

```diff
@@ -109,72 +109,7 @@
 #' to danger). Finally, you can choose to override the default tooltip with
 #' `version_tooltip`.
 #'
-#' @section YAML config - navbar:
```

Comment:
the way I re-use content is constrained by the fact one cannot, at the moment, add an Rmd to an user-defined section https://roxygen2.r-lib.org/articles/rd.html#sections-3 so the Rmd fragment contains the section title and I remove it when using the same fragment in the vignette.